common/static/coffee/spec/discussion/view/discussion_user_profile_view_spec.coffee

```diff
@@ -1,57 +1,14 @@
 describe "DiscussionUserProfileView", ->
     beforeEach ->
         DiscussionSpecHelper.setUpGlobals()
-        setFixtures(
-            """
```

Comment:
Agreed! This one was especially egregious, because it didn't match the Mustache template at all. It looked like the fixture was tweaked to make it easier to test, but that made it hard to know if it would work with the real template.